### PR TITLE
Hide read only embedded entities

### LINKED ISSUE #6034: hide all "r" embedded entities from the view in all modes
https://inmanta.slack.com/archives/C045Y42N49W/p1730916907386389

5. We need to introduce something to be able to hide/ignore certain relation so that the "allocated" block is not visible because the really "break" the flow. We can use annotations for this

- we could indeed just hide all "r" relations

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

From now on Composer won't display embedded entities with modifier set to "r"

on recording allocated entity is "r"
https://github.com/user-attachments/assets/24086e48-887f-47ba-9b41-01c06fbafeba

closes #6034 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [ ] Code is clear and sufficiently documented
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: src/UI/Components/Diagram/actions.ts [hunk 1]
```diff
@@ -358,25 +358,27 @@ export function appendEmbeddedEntity(
...
-        paper,
-        graph,
-        entity,
-        entityInstance[entity.name] as InstanceAttributeModel,
-        instanceAsTable.id as string,
-        embeddedEntity.name,
-        presentedAttr,
-        isBlockedFromEditing,
-      );
+    embeddedEntity.embedded_entities
```
Comment (LukasStordeur) [inline src/UI/Components/Diagram/actions.ts:361]: Can you please add to the comments in the code the reason why we are filtering these out.

**Review by LukasStordeur — COMMENTED** [review]
This is just a small portion of the code, but already, we are iterating through the same thing multiple times, or that's the impression I get. I don't have the overview picture of the code, but I get the impression we are looping over a lot, multiple times in different places, probably could be unified to keep it more optimized at some point

File: src/UI/Components/Diagram/actions.ts (hunk 1 above)
Comment (matborowczyk) [reply src/UI/Components/Diagram/actions.ts:361]: added

Comment (LukasStordeur) [reply src/UI/Components/Diagram/actions.ts:361]: de-cluttering is good but doesn't tell why we are doing this, I vaguely know it's about the alocated data, but I'd like to see an actual explanation on it why they don't need to be displayed.

**Review by LukasStordeur — APPROVED** [review]

**Pull request closed without merging**